lib/src/components/_stacks-tables.less

```diff
@@ -197,17 +142,152 @@
     }
 }
 
-//  $$  Footer: Large Row
-//  ------------------------------------------------------------------------
-.s-table--totals {
-    th,
-    td {
-        padding-top: @su12;
-        font-size: @fs-subheading;
-        font-weight: bold;
+//  ============================================================================
+//  $    LAYOUT VARIATIONS
+//  ----------------------------------------------------------------------------
+.s-table {
+    //  $$  STRIPES
+    //  ------------------------------------------------------------------------
+    &.s-table__stripes {
+        // Darken the header
+        thead th {
+            background-color: darken(@black-025,2%);
+        }
+        // Add in stripes
+        tbody tr:nth-child(even) {
+            background-color: @black-025;
+        }
+    }
+
+    //  $$  SPACING
+    //  ------------------------------------------------------------------------
+    //  --  SMALL
+    &.s-table__sm {
+        th,
+        td {
+            padding: @su4;
+        }
+    }
+    //  --  LARGE
+    &.s-table__lg {
+        th,
+        td {
+            padding: @su12;
+        }
+    }
+
+    //  $$  BORDERS
+    //  ------------------------------------------------------------------------
+    //  --  No Borders
+    &.s-table__b0 {
+        th,
+        td,
+        tr:last-of-type th,
+        tr:last-of-type td {
+            border-color: transparent;
+        }
+
+        //  This makes the border transparent, so we need to use whitespace
+        //  to achieve the same effect a 2px gray border achieves.
+        tbody + tbody {
+            border-color: transparent;
+            border-width: @su12;
+        }
+    }
+
+    //  --  Horizontal Only
+    &.s-table__bx {
+        th,
+        td {
+            border-left-color: transparent;
+            border-right-color: transparent;
+        }
+        thead th {
+            border-top-color: transparent;
+            border-bottom-color: @black-200;
+        }
+        tbody tr {
+            &:first-of-type th,
+            &:first-of-type td {
+                border-top-color: transparent;
+            }
+            &:last-of-type th,
+            &:last-of-type td {
+                border-bottom-color: transparent;
+            }
+        }
+        tfoot th,
+        tfoot td {
+            border-bottom-color: transparent;
+        }
+    }
+
+    //  $$  NO BACKGROUNDS
+    //  ------------------------------------------------------------------------
+    &.s-table__clear th {
```

Comment:
I think if `.s-table__clear` is to mean no backgrounds, we can keep the header typography consistent with the default styles. I'd only keep `background-color` here.